openvpn/service_discovery/dto/payment_method_per_bytes_test.go

```diff
@@ -0,0 +1,102 @@
+package dto
+
+import (
+	"encoding/json"
+	"github.com/mysterium/node/datasize"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"github.com/mysterium/node/money"
+)
+
+var (
+	price = money.NewMoney(0.5, money.CURRENCY_MYST)
+)
+
+func TestPaymentMethodPerBytesSerialize(t *testing.T) {
+	var tests = []struct {
+		model        PaymentMethodPerBytes
+		expectedJson string
+	}{
+		{
+			PaymentMethodPerBytes{
+				Price: price,
+				Bytes: datasize.Gigabyte,
```

Comment:
Field `Bytes` is not in Bytes anymore.
Should we rename it to `throughput` or `bitsize`? 
https://en.wikipedia.org/wiki/Data_rate_units

Reply:
What about a generic "threshold". Could use it for time as well and based on the context of the struct name you'll know what it's about.

Reply:
Maybe change Bytes to Amount and Duration to Amount? I think it would make sense to add GetAmount() method on the PaymentMethod interface.

Reply:
I liked something:
- `PaymentMethodPerBytes` -> `PaymentMethodPerData` raname struct also
- `Bytes` -> `Amount`
- or `Bytes` -> `Size`